### PR TITLE
Optimize-CI-pipeline

### PR DESCRIPTION
**Pull request by sanmai-NL**
# Description

Optimize performance and maintainability of the GitLab CI pipeline declaration.

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature/architecture support (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update
- [x] Build/CI update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Please provide hardware architecture specs and
instructions so we can reproduce.

- [ ] Test A: Hardware architecture, machine name, example/test run
- [ ] Test B: Hardware architecture, machine name, example/test run
- [ ] ...

# Checklist:

- [ ] I have run `./scripts/check-code-format.sh` and confirm my code code follows the style guidelines of variorum
- [ ] I have added comments in my code
- [ ] My changes generate no new warnings (build with `-DENABLE_WARNINGS=ON`)
- [ ] New and existing unit tests pass with my changes

Thank you for taking the time to contribute to Variorum!

### CODE REVIEW

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -6,13 +7,75 @@ stages:
...
+  - ./variorum-print-power-example
+  - cd ../../using-with-make/fortran
+  - echo -e "PWD:" ${PWD}
+  - make VERBOSE=1
+  - ./ep.W.x
+
 build-quartz-format:
+ extends:
+   - .astyle
+ interruptible: true
```
Comment (sanmai-NL) [inline .gitlab-ci.yml]: This makes GitLab CI cancel the job once a newer job (new commit on same branch) is scheduled.

**Comment by sanmai-NL** [comment]
Can you review this please?

**Comment by sanmai-NL** [comment]
@rountree Who can and will update me on this? A review or response is due ...

**Comment by sanmai-NL** [comment]
@slabasan Can you review this please?

**Comment by tpatki** [comment]
Closing this PR. We don't plan to maintain this internal file going forward.

**Comment by sanmai-NL** [comment]
@tpatki Can you please explain in contribution guidelines what contributions you do seek? You've adopted an OSS development model after all. This mismatch has caused a waste.

**Pull request closed without merging**